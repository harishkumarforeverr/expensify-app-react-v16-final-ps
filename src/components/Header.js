import React from 'react';
import{NavLink} from "react-router-dom";

const Header=()=>{
    return (
        <header> 
         <h1>Expensify-App</h1> 
         <NavLink exact={true} activeClassName="is-active" to="/">DashBoard</NavLink>
         <NavLink activeClassName="is-active" to="/create">Create</NavLink>
        </header>
    )
}

export default Header;