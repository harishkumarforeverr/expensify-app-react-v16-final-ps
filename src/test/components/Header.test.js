import React from 'react';
import{shallow} from 'enzyme';
import Header from "../../components/Header";

test(' Header from "../../components/Header"',()=>{ 
    const wrapper=shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
})