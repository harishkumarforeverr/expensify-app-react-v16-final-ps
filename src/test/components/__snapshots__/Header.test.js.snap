// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Header from "../../components/Header" 1`] = `
<header>
  <h1>
    Expensify-App
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    DashBoard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    Create
  </NavLink>
</header>
`;
